catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -167,10 +177,31 @@ class ValidationParserSpecTest extends Specification {
 
         then:
         thrown(IllegalArgumentException)
-        attributeRegistry.getAttributeDescriptor("cool-attribute").isPresent()
+        attributeRegistry.lookup("cool-attribute").isPresent()
         attributeValidatorRegistry.getValidators("cool-attribute").size() == 0
     }
 
+    def "test injections"() {
+        setup:
+        file.withPrintWriter { it.write(injections) }
+
+        when:
+        validationParser.install(file)
+
+        then:
+        verifyInjections()
+    }
+
+    void verifyInjections() {
```

Comment:
imo not worth factoring out this for 2 attributes + 2 asserts.

also groovy has list literals. no need for Collections. and also implicit properties if they have setters and getters. 
```groovy
assert injectableAttributeRegistry.injectableAttributes(BASIC_METACARD.name) == [coolAttribute.name]
assert injectableAttributeRegistry.injectableAttributes("nitf") == [coolAttribute, geospatialGoodness].*name
```

also can do cool stuff like
```groovy
injectableAttributeRegistry.with {
    assert injectableAttributes(BASIC_METACARD.name) == [coolAttribute.name]
    assert injectableAttributes("nitf") == [coolAttribute, geospatialGoodness].*name // <- this is spread operator
}
```

could also do a where clause which might look clean as well (eg, https://github.com/codice/alliance/pull/71/files#diff-e8822ac7ef10916a0a20f84d3b566f5dR64) though with only two idk if its worth a where